### ISSUE TITLE
USER ISSUE: 

### ISSUE THREAD

**Issue by Greatzeus1** [StrangeLoopGames/EcoIssues#5605]
**Version:** 0.7.2.1 beta

**Steps to Reproduce:**
Tried to pick up log that was cut down	

**Expected behavior:**
should pick up	

**Actual behavior:**
The following error message popped up:
```Caught exception:

<size=60.00%>Caught exception invoking RPC PlayerInteract on Player!

Object reference not set to an instance of an object.


Stack:
   at TreeEntity.PickupLog(Player player, Guid logID)
   at TreeEntity.OnActInteract(InteractionContext context)
   at Eco.Gameplay.Interactions.InteractionExtensions.ExecuteUntilSuccess(IEnumerable`1 results)
   at Eco.Gameplay.Players.Player.PlayerInteract(InteractionInfo info)



Stack:
   at Eco.Shared.Networking.RPCManager.TryInvoke(Object controller, String methodname, BSONObject bsonArgs, Object& result)
   at Eco.Shared.Networking.RPCManager.InvokeOn(BSONObject bson, Object controller, String methodname)
   at Eco.Shared.Networking.RPCManager.HandleReceiveRPC(INetClient client, BSONObject bson)
   at Eco.Shared.Networking.NetPort.ProcessBuffer()
   at Eco.Shared.Networking.NetPort.Update()
   at Eco.Plugins.Networking.Clients.Client.<.ctor>b__56_0()
</size>
Caught exception:

<size=60.00%>Caught exception invoking RPC PlayerInteract on Player!

Object reference not set to an instance of an object.


Stack:
   at TreeEntity.PickupLog(Player player, Guid logID)
   at TreeEntity.OnActInteract(InteractionContext context)
   at Eco.Gameplay.Interactions.InteractionExtensions.ExecuteUntilSuccess(IEnumerable`1 results)
   at Eco.Gameplay.Players.Player.PlayerInteract(InteractionInfo info)



Stack:
   at Eco.Shared.Networking.RPCManager.TryInvoke(Object controller, String methodname, BSONObject bsonArgs, Object& result)
   at Eco.Shared.Networking.RPCManager.InvokeOn(BSONObject bson, Object controller, String methodname)
   at Eco.Shared.Networking.RPCManager.HandleReceiveRPC(INetClient client, BSONObject bson)
   at Eco.Shared.Networking.NetPort.ProcessBuffer()
   at Eco.Shared.Networking.NetPort.Update()
   at Eco.Plugins.Networking.Clients.Client.<.ctor>b__56_0()
</size>
```

**Comment by MarjieAVolk** [comment]
Fixed in 7.2

**Comment by Greatzeus1** [comment]
No worries.  Thanks

Sent from my iPhone

On 8 Mar 2018, at 00:50, MarjieAVolk <notifications@github.com<mailto:notifications@github.com>> wrote:


I'm sorry, I was catching up on my backlog, so I didn't notice this was filed after the 7.2 release. This bug has been reopened to track this issue after the 7.2 release: #5595<https://github.com/StrangeLoopGames/EcoIssues/issues/5595>
